packages/terra-core-docs/src/terra-dev-site/doc/form-select/example/combobox/Invalid.jsx

```diff
@@ -21,6 +21,7 @@ const InvalidExample = () => {
         <Combobox.Option value="violet" display="Violet" />
       </Combobox>
       {isInvalid && <span className={cx('error-text')}>Please select a color</span>}
+      <p>Required: Please select a color from above list</p>
```

Comment:
should we also set `required` prop of combobox here ?

Reply:
Added